### ISSUE TITLE
BIPM final review: number spacing in groups of 3, but don't let 1 digit hanging

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/bipm-si-brochure#194]
From BIPM Janet Miles:

Concerning the presentation of numbers… there’s another twist written explicitly in the SI Brochure, so I think we need to respect it here:

As you know, digits are presented in groups of threes, with an inseparable space between the groups of three – unless there’s a trailing digit (left all on its own), in which case it’s included (without a space) with the previous group of three

See SI Brochure Section 5.4.4:
![image002](https://user-images.githubusercontent.com/11865/190357061-7103ae61-15a0-4a5f-935b-29c90cc4cbc6.png)

--------

Simply put, if there is 1 hanging digit before the decimal marker, present the number as is:
* "1999" => "1999" (not "1 999")
* "10999" => "10 999"

If there is 1 hanging digit after the decimal marker, don't group:
* "0.1999" => "0.1999" (not "0.199 9")
* "10.1999" => "10.1999" (not "10.199 9")

So I guess:
* "1999.1999" => "1999.1999" (instead of "1 999.199 9")
* "1999999.1999999" => "1999 999.199 9999" (instead of "1 999 999.199 999 9")

This is a localization issue to be done in Metanorma for BIPM.
